scripts/lib/spreadsheet-mappings.ts

```diff
@@ -30,6 +30,7 @@ export const FIELD_MAPPINGS: { [index: string]: string[] } = {
   other_restrictions: ['Other Restrictions'],
   stacking_details: ['Stacking Details'],
   financing_details: ['Financing Details'],
+  editorial_notes: ['Editorial Notes'],
```

Comment:
Are we able to add a comment here, or is it clear elsewhere what goes here? Unsure the name is clear enough for folks who aren't familiar, but don't have a better idea for the name.

Reply:
I added a comment. Eventually, this should be added to the data model, which is also where the other fields are documented, but given possible evolutions to how we do that, I don't think it makes sense to do now.